admin_manual/configuration_database/db_conversion.rst

```diff
@@ -69,5 +69,6 @@ Here is a list of known old tables:
 * oc_media_songs
 * oc_media_users
 * oc_permissions
+* oc_privatedata
```

Comment:
```suggestion
* oc_privatedata - this table was later added again by the app `privatedata` (https://apps.nextcloud.com/apps/privatedata) and is safe to be removed if that app is not enabled
```

Reply:
@Yvan-Masson any news here?

Reply:
@skjnldsv I do not understand if you are waiting something especially from me or not, feel free to ask precisely if I need to do something.

Reply:
I used the "suggested change" feature of GitHub here. I guess there is a button to accept this suggested change. Then this can go in 👍 

But as this is a beta feature it is maybe not visible to you. Do you see this above @skjnldsv's reply?

![bildschirmfoto 2019-02-15 um 10 24 14](https://user-images.githubusercontent.com/245432/52847209-db96b880-310b-11e9-8cdc-b9f0218c295c.png)